### ISSUE TITLE
Empty kube config results in error parsing config file 

### ISSUE THREAD

**Issue by lukepolo** [lensapp/lens#2329]
**Describe the bug**
When having an empty kube config , it produces the error  
![image](https://user-images.githubusercontent.com/2066668/110956251-70cf0000-8318-11eb-8839-c32d2dfcebe0.png)

According to minikube devs the config is valid, you can follow in the related issue : https://github.com/kubernetes/minikube/issues/10690

**To Reproduce**
add 
```
- context:
    cluster: ""
    user: ""
  name: dev
```

**Expected behavior**
Not to show an error.

**Environment (please complete the following information):**
- Lens Version: 4.1.4
- OS: osx 11.2.3
- Installation method (e.g. snap or AppImage in Linux): brew

**~/.kube/config**
```
- context:
    cluster: ""
    user: ""
  name: minikube
- context:
    cluster: qx-dev
    extensions:
    - extension:
        last-update: Tue, 02 Mar 2021 12:13:29 EST
        provider: minikube.sigs.k8s.io
        version: v1.17.1
      name: context_info
    namespace: default
    user: qx-dev
  name: qx-dev
current-context: qx-dev
kind: Config
```

**Comment by Nokel81** [comment]
Fixed by #1078

**Comment by Nokel81** [comment]
Also duplicate of #843
